### ISSUE TITLE
Not compatible with v382.53 of the driver

### ISSUE THREAD

**Issue by LolHens** [sk1080/nvidia-kvm-patcher#10]
The 1080 Ti but it doesn't seem to be compatible with v372.54 of the nvidia driver.

I tried to modify the patches for the new driver:
41FF978804000085C0 # 361.91 - 368.39
41FF97B804000085C0 # 372.54
41FF97E804000085C0 # 382.53

for the batch script the pattern also changes
488D4424504C8D4C24444533C04889442430 # 361.91 - 368.39, 372.54
0000B908000000E8A0870100488BF84885C0 # 382.53

sadly this didn't work out for me. I still got error 43 so I assume nvidia must have changed some things to prevent this.

Maybe you will be able to find a patch for the new driver someday.

I just wanted to share this. You can close this issue if you don't need it.

**Comment by digrouz** [comment]
same issue here.. so unfortunate..

**Comment by sk1080** [comment]
I apologize for taking so long to get back to you, but it would seem that the function I was patching is ghost. I might re-debug the driver when I get a chance, but I would recommend just sticking with kvm with the usual workaround at this time. Last time I tested xen things weren't happy even though the driver loaded. Not sure about other platforms, although I think I saw some possible success stories...
